roles/openshift_hosted_facts/tasks/main.yml

```diff
@@ -8,9 +8,10 @@
 
 - name: Set hosted facts
   openshift_facts:
-    role: hosted
+    role: "{{ item }}"
```

Comment:
Not much is relying on openshift_hosted_facts at this point.  I don't think this is necessary.  If you're making changes that require this, make them so they don't if you can.  This role is one task away from extinction.

Reply:
unfortunately, I had to modify this so that it would correctly set the facts for the `logging` `loggingops` and `metrics` facts so that `openshift_persistent_volumes` had the correct information to create pvs. During my testing, before I updated this it would skip creating the PV because it didn't correctly parse out facts...